cmd/main.go

```diff
@@ -209,7 +214,18 @@ func main() {
 		}
 		log.Fatal(http.Serve(unixListener, mux))
 	} else {
-		log.Fatal(http.ListenAndServe(config.Server.Bind, mux))
+		if *tlsServerCert != "" && *tlsServerKey != "" {
```

Comment:
I don't remember how the stringFlag function works - Is it possible for these to be nil? If so perhaps add a nil check as well as empty value check to avoid a possible nil pointer dereference.

Reply:
Valid question, but sadly, the default value for the flag needs to be `string` and not `*string`, so it cannot be `nil`.
Personally, I'm always a bit torn on whether or not I like this for the flags API, but I'm afraid I cannot do anything about it either way :)

Reply:
I'm pretty sure the flags API won't `nil` these, but I suppose technically speaking, they can of course be `nil` given the `*string`, so additional `nil` check definitely won't hurt.